plugins/ledger/README.md

```diff
@@ -0,0 +1,76 @@
+![Bitcoin Cash](https://raw.githubusercontent.com/The-Bitcoin-Cash-Fund/Branding/master/Bitcoin_Cash/BCH%20Logo%20Long%20Text%20WhiteBG.png "")
+# Using Electron-Cash with Ledger Nano S
+
+Successfully tested with Ledger Nano S (1.3.1) Electron-Cash (3.1.2) and Bitcoin Cash Ledger App (1.1.5)
```

Comment:
Are you sure that the ledger app version is 1.1.5? I'm pretty sure it should be 1.1.8